test/aggregate.test.js

```diff
@@ -636,15 +636,12 @@ describe('aggregate', () => {
   });
 
   describe('sort aggregations', () => {
-    it('sorts alphabetically, uppercase first', async () => {
+    it.only('sorts alphabetically, uppercase first', async () => {
```

Comment:
Don't forget to remove the `only`
```suggestion
    it('sorts alphabetically, uppercase first', async () => {
```

Reply:
Yep. Thanks!

I still need to add new test cases, anyways.